src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -1349,6 +1349,24 @@ private static string GetFirstLineSubString(string stringToComplete, out bool ha
             return keyValuePairWithCursor;
         }
 
+        private static List<CompletionResult> CompletePropertyAssignment(MemberExpressionAst memberExpression, CompletionContext context)
```

Comment:
Why not place this method right above `TryGetCompletionsForVariableAssignment`, where it gets used?

Reply:
No particular reason. I've moved it now.